src/App.tsx

```diff
@@ -1,61 +1,136 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+
+type User = {
+  id: number;
+  name: string;
+  email: string;
+};
+
+type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user?: User | null;
+};
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+};
```

Comment:
The `onSubmit` prop is defined in the `Props` type but is not used in the `App` component. If it's not needed, consider removing it from the type definition.